doc/source/tokudb/tokudb_file_management.rst

```diff
@@ -22,7 +22,7 @@ within a directory that represents a single database. This feature is enabled
 by default.
 
 In |Percona Server| :rn:`5.7.18-14` new :variable:`tokudb_dir_cmd` variable
-has been implemented that can be used to edit |TokuDB| files.
+has been implemented that can be used to edit the |TokuDB| dictionary.
```

Comment:
Going to be a bit pedantic: In Tokutek nomenclature, a dictionary is an index or a single FT key/value FT file. The directory dictionary is what maps internal dictionary names to physical dictionary files on disk. See https://www.percona.com/blog/2016/09/27/tokudb-and-perconaft-database-file-management-part-1-of-2/ that explains the files. This new feature allows a user to manipulate the contents of the TokuDB/PerconaFT directory map.

So, if we want to be completely clear and accurate, we can reword this as "In |Percona Server| :rn:`5.7.18-14` new :variable:`tokudb_dir_cmd` variable has been implemented that can be used to edit the |TokuDB| directory mapping." and even perhaps link back to this blog post or some other doc content that has the same info.